webpack.config.js

```diff
@@ -91,6 +94,11 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 } else {
 	webpackConfig.entry.vendor = [ 'react', 'store', 'page', 'wpcom', 'jed', 'debug' ];
 	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( 'vendor', '[name].[hash].js' ) );
+	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( {
```

Comment:
From: https://webpack.github.io/docs/list-of-plugins.html#commonschunkplugin
```
 If [options.name or options.names is] omitted and options.async or options.children is set all chunks are used, otherwise options.filename is used as chunk name.
```